karaf/apache-brooklyn/src/main/resources/etc/org.ops4j.pax.logging.cfg

```diff
@@ -0,0 +1,46 @@
+################################################################################
```

Comment:
do we need this file?  we already have a sophisticated `log4j.xml` config hierarchy as per website.  what is relationship of this with that?  if we keep this it needs to be explained, and maybe docs updated.